sound/soc/sof/intel/hda-ipc.c

```diff
@@ -287,7 +284,7 @@ irqreturn_t hda_dsp_ipc_irq_thread(int irq, void *context)
 		 * This interrupt is not shared so no need to return IRQ_NONE.
 		 */
 		dev_dbg_ratelimited(sdev->dev,
-				    "nothing to do in IPC IRQ thread\n");
+						"nothing to do in IPC IRQ thread\n");
```

Comment:
Not sure why, but all of these alignment changes looks like incorrect? Can you drop these?